R/prep_data_executive_summary.R

```diff
@@ -237,14 +239,29 @@ prep_data_executive_summary <- function(investor_name,
       entity_type = "benchmark"
     )
 
+  emissions_portfolio <- emissions_portfolio %>%
+    dplyr::mutate(entity = "portfolio")
+
+  emissions_indices_eq <- readr::read_rds(file.path(survey_dir, "Indices_equity_emissions.rds")) %>%
+    dplyr::mutate(entity = "benchmark")
```

Comment:
same here... you want `entity` to be filled with "benchmark", right?

Reply:
correct